standalone/runtime/src/lib.rs

```diff
@@ -301,7 +301,7 @@ parameter_types! {
 #[cfg(not(feature = "integration-tests"))]
 parameter_types! {
   // How often we trigger a new session.
-  pub const Period: BlockNumber = MINUTES;
+  pub const Period: BlockNumber = 20 * MINUTES;
```

Comment:
Are we going to leave this 20min session? shall we make it 1 hour session again? (~600 block)

Reply:
```suggestion
  pub const Period: BlockNumber = 1 * HOURS;
```